data/hai/hai reveal 4 middle.txt

```diff
@@ -1417,30 +1417,30 @@ mission "Hai Reveal [A14] A Time of Trouble"
-			`	"They haven't outright said it... but they've implied that the Navy could end up killing the Hai involved in the process by not understanding the capabilities of the Hai, or by placing too great an emphasis on preserving Navy lives. Enough people in the Cabinet disagree that we're sort of at an impasse, for the moment."`
+			`	"They haven't outright said it... but they've implied that the Navy could end up killing the Hai involved in the process by not understanding the capabilities of the Hai, or by placing too great an emphasis on preserving Navy lives. Enough people in the Cabinet disagree, that we're sort of at an impasse, for the moment."`
 			`	"Who in the Cabinet is blocking it?" The speaker is Turner, and everyone turns to look at him with a mixture of irritation or surprise.`
-			`	Xilin proceeds to list of a number of names and after a few turner cuts in again. "Remington, I know you're the PR guy and you keep your nose out of sales so I apologize if it's presumptuous for me to assume you haven't considered this but, most of those names are people who listen to the Syndicate. I should know, I've had to negotiate with them as a block often enough for government contracts. Has anyone on your side reported back to the Board and suggested that perhaps, if we can influence them to support a solution that we propose, even if we don't have that solution defined yet, the Hai could be a very large and very profitable market for Syndicate goods?"`
-			`	There is a pregnant pause as Remington looks to Terry and they engage in some kind of silent exchange before he replies. "You know, I don't think we'd connected those two specific dots just yet." He looks at Terry. "We need to make some calls."`
+			`	Xilin proceeds to list of a number of names and after a few Turner cuts in again. "Remington, I know you're the PR guy and you keep your nose out of sales, so I apologize if it's presumptuous for me to assume you haven't considered this, but, most of those names are people who listen to the Syndicate. I should know, I've had to negotiate with them as a bloc often enough for government contracts. Has anyone on your side reported back to the Syndicate Board and suggested that perhaps, if we can influence them to support a solution that we propose, even if we don't have that solution defined yet, the Hai could be a very large and very profitable market for Syndicate goods?"`
+			`	There is a long pause as Remington looks to Terry and they engage in some kind of silent exchange before he replies. "You know, I don't think we'd connected those two specific dots just yet." He looks at Terry. "We need to make some calls."`
```

Comment:
```suggestion
			`	There is a long, expectant pause as Remington looks to Terry and they engage in some kind of silent exchange before he replies. "You know, I don't think we'd connected those two specific dots just yet." He looks at Terry. "We need to make some calls."`
```

This'll preserve the meaning while still avoiding the word that most won't recognize as appropriate to the context.